src/vectorFeature.js

```diff
@@ -2,11 +2,47 @@ var inherit = require('./inherit');
 var feature = require('./feature');
 
 /**
- * Create a new instance of class vectorFeature
+ * Object specification for a graph feature.
+ *
+ * @typedef {geo.feature.spec} geo.vectorFeature.spec
+ * @property {geo.vectorFeature.styleSpec} [style] Style object with default
+ *   style options.
+ * @property {function|object} [origin={x: 0, y: 0, z: 0}] Origin accessor.
+ * @property {function
```

Comment:
looks like something is missing here

Reply:
The line shouldn't of been there.  I've removed it.

Reply:
thanks